requirements/dev-requirements.txt

```diff
@@ -15,7 +15,7 @@ fixture
 sniffer
 psutil  # for memory profiling
 django-extensions
-ipython==4.2.1
+ipython==5.2.2
```

Comment:
Are you sure?  I just tried it out and it works for me.  Could be a mac vs linux thing.  If that's the case, then yeah, we'll probably have to go with 4.x